pkg/vm/bigint.go

```diff
@@ -0,0 +1,136 @@
+package vm
+
+import (
+	"encoding/binary"
+	"math/big"
+	"math/bits"
+)
+
+// wordSizeBytes is a size of a big.Word (uint) in bytes.`
+const wordSizeBytes = bits.UintSize / 8
+
+// bytesToInt converts data in little-endian format to
+// an integer.
+func bytesToInt(n *big.Int, data []byte) *big.Int {
```

Comment:
Why `n` parameter? All users of it allocate a new `big.Int` and that could easily be done inside `bytesToInt`. It also makes these functions more symmetric.

Reply:
Yeah, I was thinking about scenarios where it is preallocated, but anyway right now I allocate new big.Int on every usage.